app/lib/view/core/information_display/MealMainEntry.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:app/view/core/icons/MealIcon.dart';
+import 'package:app/view_model/repository/data_classes/meal/Meal.dart';
+import 'package:app/view_model/repository/data_classes/settings/PriceCategory.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+class MealMainEntry extends StatelessWidget {
+  final Meal _meal;
+  final NumberFormat _priceFormat =
+      NumberFormat.currency(locale: 'de_DE', symbol: '€');
```

Comment:
does it make sense to not hard code the locale or at least store it in a global context?